src/js/langs/pt_BR.js

```diff
@@ -0,0 +1,50 @@
+/**
+* @fileoverview I18N for Português
+* @author Victor Magalhães <https://github.com/vmagalhaes>
+*/
+import i18n from '../i18n';
+
+i18n.setLanguage(['ptBR', 'pt_BR'], {
```

Comment:
I looked up the [language code](http://www.lingoes.net/en/translator/langcode.htm). There are three kinds of Portugal.
![lang-code](https://user-images.githubusercontent.com/18183560/65228853-2e06e000-db06-11e9-82ea-c25d6dc976a1.png)

I think only one code needs to be registered below.
```js
i18n.setLanguage('pt_BR', { ... });
```